include/elf/common.h

```diff
@@ -354,6 +354,10 @@
 #define EM_65816	257	/* WDC 65816/65C816 */
 #define EM_LOONGARCH	258	/* LoongArch */
 #define EM_KF32		259	/* ChipON KungFu32 */
+#define EM_U16_U8CORE	260	/* LAPIS nX-U16/U8 */
+#define EM_TACHYUM	261	/* Tachyum */
+#define EM_56800EF	262	/* NXP 56800EF Digital Signal Controller (DSC) */
```

Comment:
not microblaze related

Reply:
In alignment with upstream bintuils
https://github.com/bminor/binutils-gdb/blob/95385060771b0cac95a39320c44eca857fb177ae/include/elf/common.h#L360C1-L362C73
Keeping this in.